types/node-fetch/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for node-fetch 2.1
+// Type definitions for node-fetch 3.0
```

Comment:
```suggestion
// Type definitions for node-fetch 2.3
```

Reply:
Thanks for mentioning the version earlier, I totally omitted to check this change.